src/microbe_stage/editor/tooltips/SelectionMenuToolTip.cs

```diff
@@ -334,7 +334,7 @@ private void UpdateMpCost()
         if (mpLabel == null)
             return;
 
-        mpLabel.Text = mpCost.ToString(CultureInfo.CurrentCulture);
+        mpLabel.Text = Math.Round(mpCost, 3).ToString(CultureInfo.CurrentCulture);
```

Comment:
This doesn't use that fancy math utils rounding. I think it maybe should, or?

Reply:
Fixed.